packages/react/src/reconciler.ts

```diff
@@ -112,6 +114,24 @@ const reconciler = reactReconciler<
       }
     }
 
+    // eslint-disable-next-line guard-for-in
+    for (const key in updateProps) {
+      const element = updateProps[key] as any;
+      if (!isValidElement(element)) continue;
+
+      let fragmentContainer:
+        | FragmentContainer
+        | undefined = (instance.props as any)[key];
+      if (isFragmentContainer(fragmentContainer)) {
+        delete updateProps[key];
+      } else {
+        fragmentContainer = createFragmentContainer(root);
+        updateProps[key] = fragmentContainer;
+      }
+      const {container} = fragmentContainer;
+      reconciler.updateContainer(element, container, null, () => {});
```

Comment:
This will trigger another updates to existing (or new if not exist) fragment whenever the sub tree is updated. What I can improve is that instead of returning raw container from fragmentContainer, I can return a method allowing to update element.